charts/kube-prometheus-stack/values.yaml

```diff
@@ -1336,6 +1336,10 @@ kube-state-metrics:
       ##
       proxyUrl: ""
 
+      ## Override serviceMonitor selector
```

Comment:
Please remove. 
We don't need to add this to the variable here if we don't change the default.

Reply:
removed